elizabeth/core/elizabeth.py

```diff
@@ -551,6 +551,74 @@ def pin(self, mask='####'):
         """
         return self.custom_code(mask=mask)
 
+    @staticmethod
+    def cpf(with_mask=True):
+        """
+        Get a random CPF (brazilian social security code)
+        :param with_mask: use CPF mask (###.###.###-##) in the return
+        :returns: Random CPF
+        """
```

Comment:
It would be like that:

```
 def cpf(with_mask=True):
         """
         Get a random CPF (brazilian social security code)
         :param with_mask: use CPF mask (###.###.###-##) in the return
         :returns: Random CPF
         :Example:
              Example here.
         """
```